modules/ROOT/pages/_partials/install-mule-runtime-versions.adoc

```diff
@@ -0,0 +1,64 @@
+= Install Mule Runtime Engine in Studio
+
+// tag::intro[]
+Anypoint Studio bundles the latest Runtime engine version available for each release.
+
+If you choose to use a different Mule runtime engine version than the one bundled in your Studio distribution, you can install any compatible Mule runtime version.
```

Comment:
Maybe we can put all of this in one sentence, and add a reason that the audience might want to use another version?

"Although your Anypoint Studio distribution automatically includes the latest Mule runtime engine (Mule) version, you might want to use another compatible Mule version, for example, if you...."

OR

Because your Anypoint Studio distribution automatically includes the latest Mule runtime engine (Mule) version, install Mule in Studio only if you want to use another compatible Mule version, for example, if you...."